src/Tribe/Views/V2/Hooks.php

```diff
@@ -864,4 +867,42 @@ public function filter_customizer_sections( $sections, $customizer ) {
 
 		return $this->container->make( Customizer::class )->filter_sections( $sections, $customizer );
 	}
+
+	/**
+	 * Filters the Global Elements section CSS template to add Views v2 related style templates to it.
+	 *
+	 * @since TBD
+	 *
+	 * @param string                     $css_template The CSS template, as produced by the Global Elements.
+	 * @param Tribe__Customizer__Section $section      The Global Elements section.
+	 * @param Tribe__Customizer          $customizer   The current Customizer instance.
+	 *
+	 * @return string The filtered CSS template.
+	 */
+	public function filter_global_elements_css_template($css_template, $section, $customizer  ) {
```

Comment:
* :no_entry_sign: No space after opening parenthesis is prohibited
* :no_entry_sign: Expected exactly one space before closing parenthesis; "  " found.
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space after the first parenthesis

<sub>hash: 488062661f66e37e0fd7dc94b3d9a276</sub>